examples/nodejs-express-launchdarkly/lib/launchdarkly.js

```diff
@@ -0,0 +1,37 @@
+import * as ld from "@launchdarkly/node-server-sdk";
+
+// Initialize LaunchDarkly client
+const client = ld.init(process.env.LAUNCHDARKLY_SDK_KEY);
+
+export const getArcjetConfig = async () => {
+  // Wait for the LaunchDarkly client to be initialized
+  await client.waitForInitialization({ timeout: 5000 });
```

Comment:
You might want a tighter timeout to be more inline with arcjet SDK (500ms in prod, 1000ms in dev)